src-gui/src/main/java/org/openkilda/controller/SwitchController.java

```diff
@@ -277,8 +273,8 @@ public class SwitchController {
      * Configure switch port.
      *
      * @param configuration the configuration
-     * @param switchId the switch id
-     * @param port the port
+     * @param switchId      the switch id
```

Comment:
not sure if "@param switchId the switch id" make any sense. it is not your code, but probably would be better to remove obvious statements

Reply:
thanks, fixed